src/web/handlers/PanGestureHandler.ts

```diff
@@ -246,9 +246,16 @@ export default class PanGestureHandler extends GestureHandler {
     }
   }
 
-  protected onPointerUp(event: AdaptedEvent): void {
+  protected onPointerUp(
+    event: AdaptedEvent,
+    sourceEvent?: TouchEvent | PointerEvent
+  ): void {
     super.onPointerUp(event);
 
+    if (sourceEvent?.cancelable) {
```

Comment:
Don't other handlers (pinch, rotate) also need something like that?

Reply:
I thought that they will need it, however testing on `Transformations` example showed that it is not necessary.

Reply:
I don't think there's a scroll view in that example

Reply:
There isn't but I modifed it so it is similar to Draggable, i.e:

```jsx
export default function Example() {
  return (
    <ScrollView style={styles.home}>
      <LoremIpsum />
      <Photo />
      <LoremIpsum />
    </ScrollView>
  );
}
```